Ghidra/Processors/PowerPC/data/languages/ppc_instructions.sinc

```diff
@@ -3850,6 +3961,8 @@ define pcodeop lswxOp;
 	EA:$(REGISTER_SIZE) = RA_OR_ZERO + B;
 	if (RESERVE == 0) goto inst_next;
 	*[ram]:8 EA = storeDoubleWordConditionalIndexed(S,RA_OR_ZERO,B);
+	# set when a stwcx. or stdcx. successfully completes
+	setCrBit(cr0, 2, 1);
```

Comment:
According to the manual, `CR0 = 00b || 1 || xer_so`. I think this could be emulated here properly by using the macro `cr0flags(0);` using 0 here will clear the LT and GT flags, set the 0 flag, and copy xer_so to the fourth bit.

Reply:
@GhidorahRex   Thanks, that's much cleaner.  I did have to extend the size 0 looks like,  just did `$(REGISTER_SIZE)`